src/components/layout/Section.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import clsxm from '@/lib/clsxm';
+
+export type VariantType = {
+  [key: string]: string[];
+};
+
+type SectionType = {
```

Comment:
rename all of component props type to {componentName}Props => add a convention for all props type